infra/modules/network/main.tf

```diff
@@ -35,3 +35,65 @@ module "aws_vpc" {
   flow_log_destination_arn         = aws_cloudwatch_log_group.flow_log.arn
   flow_log_cloudwatch_iam_role_arn = aws_iam_role.vpc_flow_log_cloudwatch.arn
 }
+
+####################
+# SECURITY SUBNETS #
+####################
+
+# The configuration below creates "security subnets" with are used exclusively for
+# MicroHealth's security tooling. They were created by MicroHealth's request to confirm
+# to their standard architecture.
+
+# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/subnet
+resource "aws_subnet" "security_public" {
+  vpc_id                  = module.aws_vpc.vpc_id
+  availability_zone       = local.availability_zones[0]
+  cidr_block              = "10.${var.second_octet}.13.0/24"
```

Comment:
Above we used third octets 0 to 2, 5 to 7, and 10 to 12. Should we keep that pattern of 5s and use 15 here (and 16 below)?

That way if for example we switch to 4 azs we'll be able to keep the subnets of each kind together.

Reply:
Oh, nice catch! Changing that now

Reply:
Didn't work. I got this:

> InvalidSubnet.Range: The CIDR '10.0.16.0/24' is invalid.

Reply:
OH! The VPC cidr is only `10.0.0.0/20`. That's... small. Probably my own fault 🤷🏼 but too late to do anything about it now.